.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] As mentioned by the previous reviewer, the `.gitignore` file should contain a `test.md` file. You should add the `test.md` file and then ignore it in the `.gitignore` file but your `test.md` should be present in your local machine and not present in your repository because you ignore it in the `.gitignore` file. Please kindly fix that 🙃 🙃

Reply:
Done adding test.md file